actionpack/test/dispatch/routing_test.rb

```diff
@@ -75,16 +75,6 @@ def test_login_redirect
     verify_redirect "http://www.example.com/login"
   end
 
-  def test_logout_redirect_without_to
-    draw do
-      get "account/logout", to: redirect("/logout"), as: :logout_redirect
-    end
-
-    assert_equal "/account/logout", logout_redirect_path
-    get "/account/logout"
-    verify_redirect "http://www.example.com/logout"
```

Comment:
Why remove this?  I thought we could still support mapping to a `to:` route.

Reply:
Because the test was updated erroneously in https://github.com/rails/rails/pull/52422. It should've been removed.